versioned/dynamodb/src/test/java/com/dremio/nessie/versioned/impl/condition/TestExpressions.java

```diff
@@ -29,7 +29,7 @@
   private final Entity av0 = Entity.ofBoolean(true);
   private final Entity av1 = Entity.ofBoolean(false);
   private final Entity av2 = Entity.ofString("mystr");
-  private final Entity set1 = Entity.ofStringSet("foo", "bar");
+  private final Entity set1 = Entity.ofList(Entity.ofString("foo"), Entity.ofString("bar"));
```

Comment:
I'm pretty sure this variable is used in various parts of the tests...

Reply:
It is - I altered it to be a list to ensure the tests would pass. There are 3 affected tests.

Reply:
The use of set was used in places that require a set in dynamo. You need to remove the use entirely.